Makefile

```diff
@@ -1,10 +1,23 @@
+.POSIX:
+
 VERCMD  ?= git describe --tags 2> /dev/null
 VERSION := $(shell $(VERCMD) || cat VERSION)
+CC       = cc
```

Comment:
`CC` is `c99` (the standard POSIX C compiler) by default in POSIX make (when using `.POSIX:`). This line simply makes it impossible to use this Makefile with a different compiler without editing the Makefile. Why do you need it?